### ISSUE TITLE
Does MS1M Arcface or iBug exclude the overlapped data from LFW

### ISSUE THREAD

**Issue by lymanblue** [deepinsight/insightface#566]
Thank you for your great work!

Issue #24 shows that MS1M Arcface already remove the overlapped data from FaceScrub.
We would like to know the overlapped data of LFW is also removed or not.

Thank you.

**Comment by nttstar** [comment]
No.

**Comment by lymanblue** [comment]
From the experimental results in InsightFace, the 99.83% accuracy in LFW dataset is trained in MS1M-V2. 

I am new to face recognition area.
I just wondering the training data MS1M-V2 (Arcface or IBUG?) in the paper removes the overlapped data with LFW or not.

Thank you.

**Comment by lymanblue** [comment]
Hi~

Is dataset MS1M-IBUG the same with the cleaned MS1M dataset from the trillionpairs in Deep Glint?
The cleaned MS1M dataset states that they already exclude the overlapped data with LFW.

Thank you.

**Comment by nttstar** [comment]
Hi, the answer for both of your questions is No.

**Comment by lymanblue** [comment]
Hi~ 

I originally thought MS1M-IBUG is the same with the cleaned MS1M dataset from Deep Glint.
It turns out that they are not the same.

So, what is the difference among MS1M-IBUG, MS1M-ArcFace and the cleaned MS1M?

Thank you.

**Comment by jetsmith** [comment]
@lymanblue did you figure out about the difference between these three datasets,  is it possible to merge MS1M-ArcFace and glint dataset(including asian and msceleb-v1c)?
